internal/admin/server.go

```diff
@@ -28,6 +34,12 @@ func init() {
 	Mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
 
 	Mux.Handle("/metrics", promhttp.Handler())
+
+	// Unregister the default GoCollector.
+	prometheus.Unregister(collectors.NewGoCollector())
```

Comment:
:no_bell: why do we have to do this :sigh:, why isn't that the default?

Reply:
Because the default registry includes some default settings for Go metrics to make it easy for end users.

Customizing the default registry is a bit awkward. We could avoid this by completely eliminating use of the default registry in baseplate.go. But, that means we would need to have user switch to `promauto.With(reg)...`.

So, we do a little bit of dance to get around it.

Reply:
we are actually already using `promauto.With(reg)` because of https://github.com/reddit/baseplate.go/commit/f68284f21f52d479de33e969eb09b3270cf1dfa5

Reply:
Yea, I'm not sure that was the right thing to do. :-/